release_notes/370-release-notes.adoc

```diff
@@ -0,0 +1,185 @@
+
+[id="new-features-370"]
+== New features and enhancements
+
+[id="verify-image-signatures-rn"]
+=== Verifying image signatures against Cosign public keys
+
+You can use {product-title-short} to ensure the integrity of the container images in your clusters by verifying image signatures against pre-configured keys. You can also create policies to block unsigned images and images that do not have a verified signature and enforce the policy by using an admission controller to stop unauthorized deployment creation. Cosign key signature verification is supported. See _Verifying image signatures_ in the {product-title-short} Operating documentation for more information.
+
+[id="integration-ecs-rn"]
+=== Autogeneration of Amazon ECR registry integrations
+
+Registry integrations for Amazon Elastic Container Registry (ECR) are now auto-generated for Amazon Web Services (AWS) clusters. This feature requires that the nodes' Instance Identity and Access Management (IAM) Role has been granted access to ECR. You can turn off this feature by disabling the EC2 instance metadata service in your nodes. See _Amazon ECR integrations_ in the {product-title-short} Integrating documentation for more information.
+
+[id="identify-missing-policies-rn"]
+=== Identifying missing Kubernetes network policies
+
+Kubernetes network policies are vital in helping to enable zero-trust networking within a cluster. They reduce the impact of network attacks by limiting the opportunity for lateral movement. By default, Kubernetes resources are not isolated. Applying network policies is a recommended best practice left to the end user.
+
+{product-title-short} 3.70 ships with a new default policy that allows you to easily identify deployments that are not restricted by any ingress network policy and to trigger violation alerts accordingly.
+
+- The default policy is named *Deployments should have at least one ingress Network Policy*. It is disabled by default.
+
+- This default policy uses a new policy criterion called *Alert on missing ingress Network Policy*.
+
+- To identify pod isolation gaps, you can clone this default policy or create a new one by using the policy criterion and enabling it on selected resources.
+
+[id="spring-vulnerabilities"]
+=== Identifying Spring critical vulnerabilities
+
+A policy to detect Spring Cloud Function RCE vulnerability (CVE-2022-22963) and Spring Framework Spring4Shell RCE vulnerability (CVE-2022-22965) has been added. It is associated with the Critical severity and is enabled by default.
+
+[id="allow-privilege-escalation-rn"]
+=== Improved validation of pod security context
+
+A new policy criterion has been added to validate the value of `allowPrivilegeEscalation` within the Kubernetes security context. You can use this policy criterion to provide alerts when a deployment is configured to allow a container process to gain more privileges than its parent process.
+
+[id="admin-credentials-link-ocp"]
```

Comment:
(not a problem, but the convention on other features seems to have these end in `-rn`)